src/App.tsx

```diff
@@ -88,7 +88,13 @@ function App() {
   return (
     <main>
       {colorBox.map((x: IColorBox) => {
-        return <ColorBox onClick={() => onClick(x)} bg={x.color} key={x.id} />;
+        return(
+          <>
```

Comment:
there is no need of this fragment

Reply:
@mankinchi Did you check the issue?

Reply:
yes I have. You can simply return a ColourBox and the div in it. No need to include `<>` around the `ColorBox`

Reply:
Oh okay! Thanks for pointing it out 😄